context-monitoring/src/test/java/de/atb/context/monitoring/models/TankRefillingMonitoringData.java

```diff
@@ -57,7 +57,6 @@
     private Date monitoredAt;
     private FileSystemDataSource dataSource;
     private String monitoringDataVersion = Version.MONITORING_DATA.getVersionString();
-    @Id
```

Comment:
This needs to be kept. Otherwise the persistence with JenaBeans might not work as expected.

Reply:
re-added annotation